psd-web/app/models/team.rb

```diff
@@ -38,8 +38,12 @@ def self.load_from_keycloak(teams = KeycloakClient.instance.all_teams(Organisati
     self.ensure_names_up_to_date
   end
 
+  def self.get_assignees
```

Comment:
```suggestion
  def self.with_organisation
```
...might be a more descriptive name? (Feels a bit unconventional to use `get` within a method name in Ruby.)

Reply:
Thanks, I was a bit uncomfortable with the naming as well.